README.md

```diff
@@ -131,3 +131,8 @@ Characters firstTagCharacters(Characters source) {
 [String]: https://api.dart.dev/stable/2.0.0/dart-core/String-class.html	"String class"
 [Uint16List]: https://api.dart.dev/stable/2.0.0/dart-typed_data/Uint16List-class.html	"Uint16List class"
 [Uint8List]: https://api.dart.dev/stable/2.0.0/dart-typed_data/Uint8List-class.html	"Uint8List class"
+
```

Comment:
I'm meh on the update to the readme. It ONLY affects managers of the package, but it creates noise on every pub.dev page. I think we should drop it!

Reply:
I can drop it. When contributing publishing automation to a mono-repo we can document on the main readme; it'll be helpful for people to better understand the workflow for contributing. And I'm less concerned now with adding repo smarts to all the individual single package repos, and instead make sure our mono-repos get all the latest automation.

Reply:
Consider having a `PUBLISHING.md` file with documentation about how to publish.
The `README.md` is for end-users, not developers.